src/app/Instance/BridgeOverview/Components/BOProcessorList.tsx

```diff
@@ -1,35 +1,88 @@
+import { useTranslation } from "@rhoas/app-services-ui-components";
+import {
+  ManagedResourceStatus,
+  ProcessorResponse,
+} from "@rhoas/smart-events-management-sdk";
+import { BODashboardSkeleton } from "./BODashboardSkeleton";
+
+import { convertProcessorsToTableItems } from "./BOUtils";
 
 export interface BOProcessorListProps {
-  processorList: DemoData[];
-  onAddingProcessor: () => void;
+  instanceId: string;
+  processorList: ProcessorResponse[] | undefined;
+  bridgeStatus: string | undefined;
+  processorsError: unknown;
+  onCreateProcessor: () => void;
 }
 
 export const BOProcessorList = (props: BOProcessorListProps): JSX.Element => {
-  const { processorList, onAddingProcessor } = props;
-  const desc =
-    "Processors use Camel DSL to filter and transform events before routing events to one or more actions";
+  const {
+    instanceId,
+    processorList,
+    bridgeStatus,
+    processorsError,
+    onCreateProcessor,
+  } = props;
+
+  const { t } = useTranslation(["smartEventsTempDictionary"]);
+
+  const itemsList = useMemo(
+    () => convertProcessorsToTableItems(processorList, instanceId),
+    [instanceId, processorList]
+  );
 
   return (
-    <Card>
-      <CardTitle>Event processing</CardTitle>
-      <CardBody>
-        {processorList.length == 0 ? (
-          <BOEmptyState
-            title={"No processors"}
-            description={desc}
-            buttonLabel={"Create processor"}
-            variant={"secondary"}
-            onButtonClick={onAddingProcessor}
-          />
-        ) : (
-          <BODashboardTableView name={"processors"} demoData={processorList} />
-        )}
-      </CardBody>
-    </Card>
+    <>
+      <Card>
+        <CardTitle>{t("processor.eventProcessing")}</CardTitle>
+        <CardBody>
+          {processorsError && (
+            <EmptyState>
+              <Title headingLevel="h2" size="md">
+                {t("common.unexpectedError")}
+              </Title>
+              <EmptyStateBody>
+                {t("instance.errors.processorsListGenericError")}
+              </EmptyStateBody>
+            </EmptyState>
+          )}
+          {!processorsError &&
+            (processorList?.length === 0 ? (
+              <BOEmptyState
+                title={t("processor.noProcessors")}
+                description={t("processor.noProcessorsDescription")}
+                createButton={{
+                  title: t("processor.createProcessor"),
+                  onCreate: onCreateProcessor,
+                  isDisabled: bridgeStatus !== ManagedResourceStatus.Ready,
+                }}
+                variant={"secondary"}
+              />
+            ) : (
+              <BODashboardTableView
+                name={t("common.processors")}
+                createButton={{
+                  title: t("processor.createProcessor"),
+                  onCreate: onCreateProcessor,
+                  isDisabled: bridgeStatus !== ManagedResourceStatus.Ready,
```

Comment:
Does we have state of BODashboardTableview in storybook when the bridge status is not ready?

Reply:
We don't have story for the Tableview when the bridge is failed, but we have it for the "BOEmptystate".

Reply:
Yes, but it is the different usecase. IIUC this code is about accesibility of actions / buttons when the user has the link (url) but the bridge is not ready (for example failed). I can set this state manually in the story book. I just want to be sure that we do not want to catch this state in our storybook.